library/src/main/java/com/leinardi/android/speeddial/FabWithLabelView.java

```diff
@@ -240,8 +249,21 @@ public void onClick(View view) {
      * @param context context.
      * @param attrs   attributes.
      */
-    private void init(Context context, @Nullable AttributeSet attrs) {
-        View rootView = inflate(context, R.layout.sd_fab_with_label_view, this);
+    private void init(Context context, @Nullable AttributeSet attrs, String fabType, boolean fabTint) {
+        View rootView;
+        if (fabType.equals("fill")) {
+            if (fabTint) {
+                rootView = inflate(context, R.layout.sd_fill_fab_with_label_view, this);
```

Comment:
These layouts just differ for the the use of the `tint` attribute and for setting the `maxImageSize`.
Please find a way to do it programmatically so that is not needed to have 3 almost identical XML layouts.